doc/rpm-plugin-audit.8

```diff
@@ -0,0 +1,17 @@
+.TH "RPM-AUDIT" "8" "28 Jan 2020" "Red Hat, Inc."
+.SH NAME
+rpm-plugin-audit \- Audit plugin for the RPM Package Manager
+
+.SH Description
+
+The plugin writes basic information about rpm transactions to the audit log - like packages installed or removed.
+
+There are currently no options with which the plugin can be configured.
+
+It can be disabled permanantly by commenting out the
+\fI%__transaction_audit\fR macro in main macros file
+(typically located at \fI/usr/lib/rpm/macros\fR) or otherwise change
```

Comment:
Couple of things that go for all the plugin manuals:
Editing the main macros file is not good advice as that's supposed to represent the factory configuration and any subsequent updates will overwrite the changes. The right way to disable these is to redefine the macro to %{nil} in a /etc/rpm/macros.* file. It could also mention --noplugins as a means to temporarily disable all plugins.

As for audit, while we're actually documenting this stuff (whee!), lets also document how to actually find the events:
```ausearch -m SOFTWARE_UPDATE``` and if we really get wild, the attatched properties.

Reply:
Added a separate rpm-plugins man page that explains all this to avoid copying the same block all over again and again.

rpm-plugin-audit is now a bit more helpful but can probably still be extended.

Reply:
Oh, if you want to elaborate on the actual messages, here's a brief descriptions of the fields:

| Field | Possible values | Description |
|--------|-----------------------|----------------|
| op    | install/update/remove | package operation |
| sw    | <software label> | name-version-release.arch of the package|
| key_enforce | 0/1 | are signatures being enforced |
| gpg_res | 0/1 | result of signature check (0 == ok, 1 == fail)
| root_dir | <root directory path> | Root directory of the operation, normally "/"
| sw_type | rpm | package format |

Reply:
Ok, formatting this was as complicated as I expected...

Reply:
Oh, I didn't really mean putting that table literally in the manual, just presenting it here in that format seemed to make sense.